packages/commercetools/theme/components/Checkout/CartPreview.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div>
+    <div class="highlighted">
+      <SfHeading
+        :level="3"
+        title="Order summary"
+        class="sf-heading--left sf-heading--no-underline title"
+      />
+    </div>
+    <div class="highlighted">
+      <SfProperty
+        name="Products"
+        :value="totalItems"
+        class="sf-property--full-width sf-property--large property"
+      />
+      <SfProperty
+        name="Subtotal"
+        :value="$n(totals.subtotal, 'currency')"
+        :class="['sf-property--full-width', 'sf-property--large', { discounted: totals.special > 0 }]"
+      />
+      <SfProperty
+        v-for="discount in discounts"
+        :key="discount.id"
+        :name="discount.name + (discount.code && ` (${discount.code})`)"
+        :value="'-' + $n(discount.value, 'currency')"
+        class="sf-property--full-width sf-property--small"
+      />
+     <SfProperty
+        v-if="totals.special > 0"
+        :value="$n(totals.special, 'currency')"
+        class="sf-property--full-width sf-property--small property special-price"
+      />
+      <SfProperty
+        name="Shipping"
+        v-if="chosenShippingMethod && chosenShippingMethod.zoneRates"
+        :value="$n(getShippingMethodPrice(chosenShippingMethod), 'currency')"
+        class="sf-property--full-width sf-property--large property"
+      />
+      <SfProperty
+        name="Total"
+        :value="$n(totals.total, 'currency')"
+        class="sf-property--full-width sf-property--large property-total"
+      />
+    </div>
+    <div class="highlighted promo-code">
+      <SfInput
+        data-cy="cart-preview-input_promoCode"
+        v-model="promoCode"
+        name="promoCode"
+        :label="$t('Enter promo code')"
+        class="sf-input--filled promo-code__input"
+      />
+      <SfButton class="promo-code__button" @click="() => applyCoupon({ couponCode: promoCode })">{{ $t('Apply') }}</SfButton>
+    </div>
+    <div class="highlighted">
+      <SfCharacteristic
+        v-for="characteristic in characteristics"
+        :key="characteristic.title"
+        :title="characteristic.title"
+        :description="characteristic.description"
+        :icon="characteristic.icon"
+        class="characteristic"
+      />
+    </div>
+  </div>
+</template>
+<script>
+
+import {
+  SfHeading,
+  SfButton,
+  SfCollectedProduct,
+  SfProperty,
+  SfCharacteristic,
+  SfInput,
+  SfCircleIcon
+} from '@storefront-ui/vue';
+import { computed, ref } from '@vue/composition-api';
+import { useCheckout, useCart, cartGetters } from '@vue-storefront/commercetools';
+import getShippingMethodPrice from '@/helpers/Checkout/getShippingMethodPrice';
+
+export default {
+  name: 'CartPreview',
+  components: {
+    SfHeading,
+    SfButton,
+    SfCollectedProduct,
+    SfProperty,
+    SfCharacteristic,
+    SfInput,
+    SfCircleIcon
+  },
+  setup() {
+    const { chosenShippingMethod } = useCheckout();
```

Comment:
I don't think we need that

Reply:
I am afraid we should not get rid of that in this PR. It should be done in the next one - `Shipping Method refactor`. There is nothing that we could use instead for now.

I am not sure if you noticed that. This component is copy-paste from the nuxt-theme-module with removed getters.